srcpkgs/havoc/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'havoc'
+pkgname=havoc
+version=0.1.0
+revision=1
+build_style=gnu-makefile
+makedepends="libxkbcommon-devel wayland-devel wayland-protocols"
+short_desc="Minimal terminal emulator for Wayland"
+maintainer="Paper <paper@tilde.institute>"
+license="Public Domain, MIT"
+homepage="https://github.com/ii8/havoc/"
+distfiles="https://github.com/ii8/havoc/archive/$version.tar.gz"
+checksum=a214f55317e394a2b3fc90d8bc7ceec497ecc9fb7a79418cff19b08ca2832b1a
+
+pre_build() {
+	cp /usr/share/wayland-protocols/stable/xdg-shell/xdg-shell.xml .
```

Comment:
It should work without copying this. Was there some problem with VPATH? Perhaps VPATH needs to be modified with += in the makefile.

Reply:
It works without this when I install pkg-config. This was an experiment if it will solve building on arm architectures. It didn't work, I am not sure why pkg-config isn't detecting xdg-shell on arm architectures.

Reply:
Although wayland-protocols is architecture independent, the files will probably still be installed in the cross-compilation dependency base path. Not sure why pkg-config can't find it there, perhaps something in my makefiles pkg-config usage is not cross-compilation friendly.

You can check with `CROSS_BUILD` if cross compilation is happening, and `XBPS_CROSS_BASE` will be the root directory relative to which xdg-shell.xml should exist.